packages/swapper/src/swappers/thorchain/constants.ts

```diff
@@ -2,3 +2,4 @@ import { bn } from '../utils/bignumber'
 // TODO: read from https://daemon.thorchain.shapeshift.com/lcd/thorchain/constants
 export const RUNE_OUTBOUND_TRANSACTION_FEE_CRYPTO_HUMAN = bn('0.02')
 export const MINIMUM_USD_TRADE_AMOUNT = bn(1)
+export const MINIMUM_USD_TRADE_AMOUNT_ETHEREUM_NETWORK = bn(42)
```

Comment:
Why is this not `50`, you ask?

Currently, `getThorTradeQuote` applies a padding (`THOR_MINIMUM_PADDING`) to the minimum amount.
This padding is currently set to `1.2`, and so if the value here was `50` then the actual minimum on the front-end would be `60`.